input/pagecontent/general-requirements.md

```diff
@@ -228,7 +228,7 @@ For mandatory elements (minimum cardinality is > 0), the element **SHALL** be po
 - where a requester may know that the data is suppressed use the code `masked` from the [DataAbsentReason Code System](http://terminology.hl7.org/CodeSystem/data-absent-reason) following the section on [Missing Data](#missing-data).
 
 ### Mandatory Elements
-Mandatory elements are elements with a minimum cardinality of 1 (min=1). When an element is Mandatory, the data is expected to always be present. Very rarely, it may not be, and in this circumstance the requirements in [Missing Data](#missing-data) **SHALL** be applied.
+Mandatory elements are elements with a minimum cardinality of 1 (min=1). When an element is Mandatory, the data is expected to always be present. Very rarely, it may not be, and in this circumstance the requirements in [Missing Data](#missing-data) **SHALL** be applied. An element can be both _Must Support_ and mandatory, in which case the requirements for mandatory's Missing data requirements apply as described in [Missing Must Support and Mandatory Data](general-requirements.html#missing-must-support-and-mandatory-data).
```

Comment:
In [Fixes for FHIR-45065 & FHIR-45102](https://github.com/hl7au/au-fhir-core/pull/167/commits/5cab0ca9b37159205233219da00c984dcb008f8e): Line 231: 

> "An element can be both _Must Support_ and mandatory, in which case the requirements for mandatory’s Missing data requirements apply.."
Consider instead: 
"An element can be both _Must Support_ and mandatory, in which case the missing data requirements for a mandatory element applies.."
to remove the inconsistent capitalisation on "Missing" and perhaps improve readability. This is a suggestion only.

Reply:
Fixes have occurred in the Must Support & Obligation branch.